pkl-certs/pkl-certs.gradle.kts

```
@@ -0,0 +1,19 @@
+plugins {
+  pklAllProjects
+  pklJavaLibrary
+  pklPublishLibrary
+}
+
+publishing {
+  publications {
+    named<MavenPublication>("library") {
+      pom {
+        url.set("https://github.com/apple/pkl/tree/main/pkl-certs")
+        description.set("""
+          Pkl's built-in CA certificates. 
+          Used by pkl-cli and optionally supported by pkl-core.")

```

Also used by pkl-doc, pkl-codegen-java, pkl-codegen-kotlin, etc.

```suggestion
          Used by Pkl's CLIs, and optionally supported by pkl-core.")

```

---

fixed